src/cider/nrepl/middleware/ns.clj

```diff
@@ -17,39 +17,51 @@
        (filter #(= (first %) name))
        (map second)))
 
-(defn ns-vars-clj [ns]
-  (->> (symbol ns)
-       ns-publics
-       keys
-       (map name)
-       sort))
-
-(defn ns-vars-with-meta-clj [ns]
-  (->> (symbol ns)
-       ns-interns
-       (u/update-vals (comp um/relevant-meta meta))
-       (u/update-keys name)
-       (into (sorted-map))))
+(defn ns-vars-clj [ns & [include-privates?]]
+  (let [fetch-vars (if include-privates?
```

Comment:
We can also use `var-query` here directly. That would simplify a bit the code.

Reply:
Thank you for reviewing!
I missed `var-query` API. I'll fix.

Reply:
We don't have it yet (something that @arichiardi can probably tackle), but I think we can still adopt the same API and just "fake" it on the cljs side for now (by ignoring everything but the private filter).

Reply:
@bbatsov OK! I fixed to take `var-query` option for filtering vars.
On the cljs side, it only support `private?` query currently.